<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>490</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="seriesDatabaseGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>465</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Series database</string>
     </property>
     <widget class="QRadioButton" name="tmdbRadioButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TMDb</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="omdbRadioButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>OMDb</string>
      </property>
     </widget>
     <widget class="QLabel" name="recommendLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>32</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#353535;&quot;&gt;(recommended)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="saveOnExitGroupBox">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>0</y>
       <width>160</width>
       <height>130</height>
      </rect>
     </property>
     <property name="title">
      <string>Save on exit</string>
     </property>
     <widget class="QCheckBox" name="savePathCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="saveSeriesCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Series</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="interfaceGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>290</width>
       <height>130</height>
      </rect>
     </property>
     <property name="title">
      <string>Interface</string>
     </property>
     <widget class="QCheckBox" name="showSeriesInformationCheckBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>95</y>
        <width>290</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show series information</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selectInterfaceLanguageComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>170</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="languageLabel">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="darkThemeCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>290</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dark theme (requires restart)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="video">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QListWidget" name="fileTypeListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>465</width>
       <height>150</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Fixed</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="gridSize">
      <size>
       <width>90</width>
       <height>18</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="videoGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>465</width>
       <height>65</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="title">
      <string>New format</string>
     </property>
     <widget class="QLineEdit" name="newFileTypeLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>200</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="newFileTypeAddButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>25</y>
        <width>120</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileTypeRemoveButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>25</y>
        <width>120</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="nameScheme">
    <attribute name="title">
     <string>Name scheme</string>
    </attribute>
    <widget class="QGroupBox" name="newNameSchemeGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>465</width>
       <height>65</height>
      </rect>
     </property>
     <property name="title">
      <string>New name scheme</string>
     </property>
     <widget class="QLineEdit" name="newNameSchemeLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>200</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="newNameSchemeAddButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>25</y>
        <width>120</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nameSchemeRemoveButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>25</y>
        <width>120</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="nameSchemeListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>465</width>
       <height>150</height>
      </rect>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Fixed</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>275</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="toolTip">
    <string>Use with caution. Resets the video formats, name schemes and all settings to default</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>275</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
